tests/container_test.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env pytest -vs
+"""Tests for example container."""
+
+# Standard Python Libraries
+import time
+
+# Third-Party Libraries
+import requests
+
+READY_MESSAGE = "Starting Domain Manager API"
+
+
+def test_container_count(dockerc):
+    """Verify the test composition and container."""
```

Comment:
The text from the skeleton isn't very good here- it should probably be changed to something like this:
```suggestion
    """Verify that the correct number of containers started."""
```

Reply:
Fixed in this commit https://github.com/cisagov/domain-manager-api/pull/194/commits/eff0647ef244b7abef7731b99e59953ca29bf239